lib/query.js

```diff
@@ -2476,7 +2477,7 @@ Query.prototype.indexWait = Query.prototype.indexStatus;
 Query.prototype.funcall = function(args, options, internalOptions) {
   var self = this;
   // FUNCALL, FN [ AR[], BODY]
-  //  0       1  [ 0 [ ...], 
+  //  0       1  [ 0 [ ...],
   util.assertArityRange(1, Infinity, args, this);
   if (args.length === 1) {
     return self.evaluate(args[0], internalOptions);
```

Comment:
This returns a promise that will be resolved as the url.
You need to also chain `self.evaluate(options, internalOptions)` to evaluate the options like `{timeout: 200}`.
And then you can call `request.get`